src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -459,12 +475,27 @@ void KeeperStorageSnapshot<Storage>::deserialize(SnapshotDeserializationResult<S
             storage.nodes_digest += node.getDigest(path);
 
         storage.container.insertOrReplace(std::move(path_data), path_size, std::move(node));
+
+        if constexpr (use_rocksdb)
+        {
+            if (batch_load_size && nodes_read % keeper_context->getCoordinationSettings()[CoordinationSetting::rocksdb_load_batch_size] == 0)
+            {
+                storage.container.commitBatch();
```

Comment:
start / commit batch from outside is strange ... how about pass the batch size to the container, and commit automatically when `batch = rocksdb_load_batch_size`